### ISSUE TITLE
Error while running executable

### ISSUE THREAD

**Issue by AloW0304** [MehulKhanna/PokeGrinder#32]
Probably something wrong I've done, however when I run the executable I get the error: error when unmarshalling fishes.json, unexpected end of JSON input.
I have not touched the fishes.json file at all, and the json file is valid when I checked it on two different websites, so I'm unsure as to what has gone wrong.

**Comment by MehulKhanna** [comment]
> Probably something wrong I've done, however when I run the executable I get the error: error when unmarshalling fishes.json, unexpected end of JSON input. I have not touched the fishes.json file at all, and the json file is valid when I checked it on two different websites, so I'm unsure as to what has gone wrong.

Can you paste the whole error here please.

**Comment by AloW0304** [comment]
inder-macos64 ; exit;
error when unmarshalling fishes.json, unexpected end of JSON input
error when unmarshalling config, unexpected end of JSON input
Starting sessions...
PokeGrinder is running, press CTRL+C to exit!

**Comment by AloW0304** [comment]
although it says its running, nothing happens

**Comment by MehulKhanna** [comment]
> inder-macos64 ; exit; error when unmarshalling fishes.json, unexpected end of JSON input error when unmarshalling config, unexpected end of JSON input Starting sessions... PokeGrinder is running, press CTRL+C to exit!

You can try creating new json files and copy pasting the content from github.

**Comment by AloW0304** [comment]
Nope, it didn't change anything. I made new JSON files for both. made sure that in main.go it is trying to read the correct file. Only thing I can think of is that I'm on a Mac, which might change something (I did download the correct executable file). I've downloaded each individual file and put the executable and the two JSON files into a separate folder inside the main folder. Tried, same error. Taken them out and back into the main folder. Tried, same error. Nothing seems to work
